pkg/config/config.go

```diff
@@ -258,3 +262,15 @@ func passwordOrTokenExistIfUsernameProvided(a *Config) []error {
 
 	return passwordErrors
 }
+
+// templateCanBeParsed tests that the message template can be parsed or returns an error
+func templateCanBeParsed(a *Config) []error {
```

Comment:
Can this just be an `error` instead of a slice?

Reply:
No, that has to be a `[]error` because it's got to match the signature of the anonymous function that's calling it as well as the other tests.

See: https://github.com/openshift/compliance-audit-router/blob/main/pkg/config/config.go#L127-L131